### PR TITLE
Add lower bounds to Multimodal dependencies

### PR DESCRIPTION
**Pull request by giswqs**
This PR adds lower bounds to multimodal dependencies, addressing issue #2901
@sxjscience 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/setup.py [hunk 1]
```diff
@@ -28,17 +28,17 @@
...
     "requests>=2.21,<3",
-    "jsonschema<4.18",
-    "seqeval<1.3.0",
-    "evaluate<0.4.0",
+    "jsonschema>=4.14,<4.18",
+    "seqeval>=1.2.2,<1.3.0",
+    "evaluate>=0.2.2,<0.4.0",
     "accelerate>=0.9,<0.17",
-    "timm<0.7.0",
+    "timm>0.6,<0.7.0",
```
Comment (sxjscience) [inline multimodal/setup.py:35]: May be do >=0.6.12 ?

File: multimodal/setup.py [hunk 2]
```diff
@@ -28,17 +28,17 @@
...
     "torch>=1.9,<1.14",
     "torchvision<0.15.0",
     "fairscale>=0.4.5,<0.4.14",
     "scikit-image>=0.19.1,<0.20.0",
     "pytorch-lightning>=1.8.0,<1.10.0",
-    "text-unidecode<1.4",
+    "text-unidecode>1.2,<1.4",
```
Comment (sxjscience) [inline multimodal/setup.py:41]: Consider change to “>=1.3”

**Comment by sxjscience** [comment]
Commented on two lower-bounds. I think we can use “>=xxxx” for the lower-bounds.

**Comment by giswqs** [comment]
I have updated the two lower bounds as suggested:
- `timm>=0.6.12,<0.7.0`
- `text-unidecode>=1.3,<1.4`

**Review by tonyhoo — APPROVED** [review]

**Comment by Innixma** [comment]
@giswqs can you rebase? Should fix unit tests

**Review by sxjscience — APPROVED** [review]
LGTM! Thanks for the fix!

**Comment by giswqs** [comment]
All checks have passed!

**Comment by sxjscience** [comment]
Thanks @giswqs ! The PR has been merged!

**Pull request merged**